content/contracts/5.x/api/crosschain.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: "Crosschain"
```

Comment:
Are there any links that point to this file? I can't find it in the UI (except for a similar one from community-contracts)

Reply:
Ah good catch, let me update the navigation tree and the index file